services/s3/src/main/resources/codegen-resources/customization.config

```diff
@@ -251,5 +251,8 @@
     "software.amazon.awssdk.services.s3.internal.handlers.ExceptionTranslationInterceptor",
     "software.amazon.awssdk.services.s3.internal.handlers.GetObjectInterceptor",
     "software.amazon.awssdk.services.s3.internal.handlers.CopySourceInterceptor"
-  ]
+  ],
+  "enabledTraitValidations": {
+    "RequiredTrait": ["software.amazon.awssdk.services.s3"]
```

Comment:
Because this is in the S3 module, do we need the package name here? Or can we just do "requiredTraitEnabled" = true?

Reply:
The idea was to eventually generalize this to all services, so this was left more as a proof-of-concept. Discussing offline.